main/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path
+from django.contrib.auth import views as auth_views
+from . import views
+
+urlpatterns = [
```

Comment:
I would move this list to be used directly at `Hotails/url.py` to improve visibility (so that you wont need to just from one file to another to understand the url and routs)

Reply:
Done.